datafusion/sqllogictest/test_files/copy.slt

```diff
@@ -538,26 +538,6 @@ select * from validate_arrow_file;
 1 Foo
 2 Bar
 
-# Copy from dict encoded values to single arrow file
```

Comment:
I don't think we should delete this test -- it is still valid to write a dictionary encoded value to a file and that should be possible to read back out